integreat_cms/cms/templates/tutorials/page_tree.html

```diff
@@ -0,0 +1,26 @@
+{% extends "../generic_tutorial_dialog.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block content %}
+<!-- Pop-Up Header -->
+<div class="flex items-center bg-blue-500 w-full rounded min-h-12">
+    <i data-feather="help-circle" class="text-white ml-2"></i>
+    <h2 class="flex items-center font-semibold leading-normal pl-2 text-white text-2xl"> {% trans 'Introduction - Page Tree' %}</h2>
```

Comment:
```suggestion
    <h2 class="flex items-center font-semibold leading-normal pl-2 text-white text-xl"> {% trans 'Introduction - Page Tree' %}</h2>
```
I think `text-2xl` is a bit large